orchestra/tests/test_machine_tasks.py

```diff
@@ -76,6 +92,12 @@ def test_already_processing_task(self):
         assignment.status = TaskAssignment.Status.PROCESSING
         assignment.save()
 
+        # Reset iteration for assignment
```

Comment:
To pretend that the machine task is still processing, since there's not a good way to stop our synchronous machine task from executing to completion!